src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1488,7 +1488,7 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             $container->register($busId, MessageBus::class)->addArgument(array())->addTag('messenger.bus');
 
             if ($busId === $config['default_bus']) {
-                $container->setAlias('message_bus', $busId);
+                $container->setAlias('message_bus', $busId)->setPublic(true);
```

Comment:
or `$container->setAlias('message_bus', new Alias($busId, true));`

Reply:
Yep. using setPublic is more explicit to me :)